metsrw/fsentry.py

```diff
@@ -63,6 +74,13 @@ def __init__(self, path, label=None, use='original', type=u'Item', children=None
         self.children = children
         self.file_uuid = file_uuid
         self.derived_from = derived_from
+        if bool(checksum) != bool(checksumtype):
```

Comment:
I was trying to check that both checksum & checksumtype were passed, or neither.  Obviously if they're both passed they won't be identical values, so I'm trying to check truthiness by converting to bool.